src/Raven.Studio/typescript/viewmodels/resources/widgets/ongoingTasksWidget.ts

```diff
@@ -1,202 +1,277 @@
 import clusterDashboard = require("viewmodels/resources/clusterDashboard");
 import websocketBasedWidget = require("viewmodels/resources/widgets/websocketBasedWidget");
+import virtualGridController = require("widgets/virtualGrid/virtualGridController");
+import virtualColumn = require("widgets/virtualGrid/columns/virtualColumn");
+import iconsPlusTextColumn = require("widgets/virtualGrid/columns/iconsPlusTextColumn");
+import textColumn = require("widgets/virtualGrid/columns/textColumn");
+import genUtils = require("common/generalUtils");
+import clusterDashboardWebSocketClient = require("common/clusterDashboardWebSocketClient");
 
-class taskNodes {
-    nodes = ko.observable<Set<string>>();
-    nodesArray: KnockoutComputed<string[]>;
-
-    constructor() {
-        this.nodes(new Set<string>());
-        this.nodesArray = ko.pureComputed(() => _.sortBy(Array.from(this.nodes())));
-    }
-
-    addNode(nodeTag: string): void {
-        const nodeSet = this.nodes();
-        nodeSet.add(nodeTag);
-        this.nodes(nodeSet);
-    }
-
-    removeNode(nodeTag: string): void {
-        const nodeSet = this.nodes();
-        nodeSet.delete(nodeTag);
-        this.nodes(nodeSet);
-    }
-}
-
-// inner table item
-class databaseTaskItem {
-    databaseName = ko.observable<string>();
-    taskCount = ko.observable<number>();
-    taskNodes = ko.observable<taskNodes>();
-    
-    constructor() {
-        this.taskNodes(new taskNodes());
+class rawTaskItem {
```

Comment:
no need for class then? It might be simple interface?

Reply:
done - changed to interface